File: EjercicioJava/Ejercicio3_1/numeroPrimo.kt

fun main() {
   for(contador in 2..500)
       numeroPrimos(contador)
}

private fun numeroPrimos(numero: Int) {

    var esPrimo = true

    for (contador in 2..(numero - 1)) {
        if (numero.mod(contador) == 0)
            esPrimo = false
    }

    if(esPrimo)
        println("numero $numero es primo")
    else
        println("numero $numero no es primo")
}

File: Pascal/pascal/numero primo.pas
Program Primo; 
uses crt; 
var 
   cont, num, i : integer; 
begin 
   write('Introduce numero '); 
   readln(num); 
   cont:=0; 
   for i:=1 to num do 
      if num mod i = 0 then 
   cont:=cont+1; 
   if cont = 2 then 
      writeln('Es primo...') 
   else 
      writeln('No es primo...'); 
   readln; 
end. 

